src/components/ActNowSummary/index.js

```diff
@@ -0,0 +1,54 @@
+import { Grid, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import PropTypes from "prop-types";
+import React, { useState, useEffect } from "react";
+
+const useStyles = makeStyles(() => ({
+  root: {
+    textAlign: "right",
+    backgroundColor: "#F7F7F7",
+    padding: "1rem",
+  },
+}));
+
+function ActNowSummary({ className, ...props }) {
+  const classes = useStyles(props);
+  const [registeredAccountsCount, setRegisteredAccountsCount] = useState(0);
+  const [petitionsCount, setPetitionsCount] = useState(0);
+
+  useEffect(() => {
+    fetch(`/api/accounts`, {}).then(async (response) => {
+      await response.json().then((data) => {
+        setRegisteredAccountsCount(data.summary.accounts.count);
+        setPetitionsCount(data.summary.petitions.count);
+      });
+    });
+  }, []);
```

Comment:
Fetch data in the `getStaticProps` of `/pages/act-now` so that the page can be created on the server.